src/ctap/hid/mod.rs

```diff
@@ -137,6 +137,7 @@ impl CtapHid {
     const TIMEOUT_DURATION: Duration<isize> = Duration::from_ms(100);
     const WINK_TIMEOUT_DURATION: Duration<isize> = Duration::from_ms(5000);
 
+    #[allow(clippy::new_without_default)]
```

Comment:
Why do you sometimes `#allow`, and sometimes implement `Default`?

Reply:
Let's just allow this lint everywhere, as we don't use it.